sdk/core/core-http/lib/restError.ts

```diff
@@ -14,13 +14,25 @@ export class RestError extends Error {
   request?: WebResource;
   response?: HttpOperationResponse;
   body?: any;
-  constructor(message: string, code?: string, statusCode?: number, request?: WebResource, response?: HttpOperationResponse, body?: any) {
+  parsedBody?: any;
```

Comment:
prefer `unknown` to `any`

Reply:
Maybe we should make a sweep over libraries as a whole to start changing this if/when we create a guideline for it?

Reply:
I agree, there are a lot of any in this library that we inherited. We should log an issue and do a clean up as part of that.

Reply:
Changed for parsedHeaders since this is newly introduced in this PR. For the body, we will clean up in the future.